<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dropbox</class>
 <widget class="QDialog" name="Dropbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dropbox - Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="group_account">
     <property name="title">
      <string>Account</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QWidget" name="widget_authorize" native="true">
        <layout class="QVBoxLayout" name="layout_login">
         <item>
          <widget class="QPushButton" name="button_authenticate">
           <property name="text">
            <string>Login with Google Account</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_loggedIn" native="true">
        <layout class="QFormLayout" name="layout_account">
         <item row="0" column="0">
          <widget class="QLabel" name="label_loggedIn">
           <property name="text">
            <string>Logged in:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_user">
           <property name="text">
            <string>label_user</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="button_logout">
           <property name="text">
            <string>Log out</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_name">
     <property name="title">
      <string>Name</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_format">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="input_nameFormat"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_resultLabel">
        <property name="text">
         <string>Result:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_example">
        <property name="text">
         <string>example</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_folder">
     <property name="title">
      <string>Folder</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_folder">
        <property name="text">
         <string>Folder:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="input_folderName">
        <property name="text">
         <string>ScreenCloud</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_clipboard">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Clipboard</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="radio_publiclink">
        <property name="text">
         <string>Copy public link</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_dontcopy">
        <property name="text">
         <string>Dont copy link</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>button_authenticate</tabstop>
  <tabstop>button_logout</tabstop>
  <tabstop>input_nameFormat</tabstop>
  <tabstop>radio_publiclink</tabstop>
  <tabstop>radio_dontcopy</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dropbox</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dropbox</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
